tests/unit/chronology/test_admin.py

```diff
@@ -1,13 +1,29 @@
 # pylint: disable=redefined-outer-name,protected-access
```

Comment:
With pytests there's neat way of handling the fixtures - you can keep them in conftest files (on different test directory levels depending on context). What do you think about moving all fixtures there?